index.html

```diff
@@ -3,19 +3,50 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | My Name Taslima (My Github user-name Taslima26)</title>
```

Comment:
Great work Tas, well done. I think there is no need to say My Github user-name Taslima26. only Taslima26 is enough. 
code wise, nothing I can add really, way better than me. keep up the good work